src/openrct2/Game.cpp

```diff
@@ -363,6 +363,12 @@ void GameLoadInit()
     auto& gameState = GetGameState();
     windowManager->SetMainView(gameState.SavedView, gameState.SavedViewZoom, gameState.SavedViewRotation);
 
+    if (gameState.CurrentlyFollowingEntity != EntityId::GetNull())
```

Comment:
I don't think this is where the code should go. It's ultimately a ui property so it should be in the ui library. Saying that i don't know where i would place it.

Reply:
Perhaps it could go in the `SetMainView` function in WindowManager.cpp?